node/utils/attributes.ts

```diff
@@ -114,7 +114,7 @@ const convertValues = (
 ): { type: FilterType; values: FilterValue[] } => {
   // When creating a filter for price attribute, it should be the only one to use
   // the type `'PRICERANGE'`.
-  if (attribute.type === 'number' && attribute.key === 'price') {
+  if (attribute.type === 'number' && ['price', 'pre-', 'precio', 'preço'].some(p => p === attribute.key)) {
```

Comment:
can you create a separate constant for this array just to improve readability?
```suggestion
  const knownPriceKeys = ['price', 'pre-', 'precio', 'preco', 'pret']

  if (attribute.type === 'number' && knownPriceKeys.some(p => p === attribute.key)) {
```